subsys/caf/modules/ble_adv.c

```diff
@@ -473,7 +470,7 @@ static void disconnect_peer(struct bt_conn *conn)
 	}
 }
 
-static void ble_initialized(void)
+static void ble_ready(void)
```

Comment:
Please avoid renames in the same commit that are not related to the commit change.

(The ```ble_ready``` name seems better because it is called in response to the READY event. However, this part worked like this before the main PR change and should be corrected in a separate commit).

Reply:
The rename is related to the change. When ble_bond was disabled we used to wait only for BLE initialization, now we properly wait for BLE readiness to start advertising (BLE becomes ready after it's initialized and settings are loaded).